extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/devui/Name.java

```diff
@@ -11,12 +11,15 @@
 import org.jboss.jandex.Type;
 import org.jboss.jandex.Type.Kind;
 
+import io.smallrye.common.annotation.SuppressForbidden;
+
 public class Name implements Comparable<Name> {
 
     public static Name from(DotName dotName) {
         return new Name(dotName.toString(), dotName.withoutPackagePrefix());
     }
 
+    @SuppressForbidden(reason = "Type.toString() is what we need here")
     public static Name from(Type type) {
         return new Name(type.toString(), createSimpleName(type));
```

Comment:
@mkouba @manovotn @Ladicek My understanding is that `type.toString()` could contain unwanted output such as annotations so I'm not sure it's the right choice here. But then going to the `name()` would drop the potential type variables info?

Reply:
`Name` seems to be used for things like beans, scopes, qualifiers etc. where type annotations are not common. (Note that `Type.toString()` only contains _type_ annotations.) So this seems good enough to me.

(Using `Type.name()` here would drop more than just type variables; it would turn `List<String>` to `List` for example.)

Reply:
> where type annotations are not common

I've never seen any type annotations in any CDI code. So I'd say it's _pretty rare_ and not _uncommon_ ;-).

> So this seems good enough to me.

+1